packages/integrations/markdoc/README.md

```diff
@@ -99,100 +99,69 @@ const { Content } = await entry.render();
 
 ### Using components
 
-You can add Astro and UI framework components (React, Vue, Svelte, etc.) to your Markdoc using both [Markdoc tags][markdoc-tags] and HTML element [nodes][markdoc-nodes].
+You can add Astro components to your Markdoc using both [Markdoc tags][markdoc-tags] and HTML element [nodes][markdoc-nodes].
 
 #### Render Markdoc tags as Astro components
 
-You may configure [Markdoc tags][markdoc-tags] that map to components. You can configure a new tag from your `astro.config` using the `tags` attribute.
+You may configure [Markdoc tags][markdoc-tags] that map to components. You can configure a new tag by creating a `markdoc.config.mjs|ts` file at the root of your project and configuring the `tag` attribute.
 
+This example renders an `Aside` component, and allows a `type` prop to be passed as a string:
 
 ```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import markdoc from '@astrojs/markdoc';
-
-// https://astro.build/config
-export default defineConfig({
-  integrations: [
-    markdoc({
-      tags: {
-        aside: {
-          render: 'Aside',
-          attributes: {
-            // Component props as attribute definitions
-            // See Markdoc's documentation on defining attributes
-            // https://markdoc.dev/docs/attributes#defining-attributes
-            type: { type: String },
-          }
-        },
-      },
-    }),
-  ],
-});
+// markdoc.config.mjs
+import { defineMarkdocConfig } from '@astrojs/markdoc/config';
+import Aside from './src/components/Aside.astro';
+
+export default defineMarkdocConfig({
+  tags: {
+    aside: {
+      render: Aside,
+      attributes: {
+      // Markdoc requires type defs for each attribute.
+      // These should mirror the `Props` type of the component
+      // you are rendering. 
+      // See Markdoc's documentation on defining attributes
+      // https://markdoc.dev/docs/attributes#defining-attributes
+        type: { type: String },
+      }
+    },
+  },
+})
 ```
 
-Then, you can wire this render name (`'Aside'`) to a component from the `components` prop via the `<Content />` component. Note the object key name (`Aside` in this case) should match the render name:
+This component can now be used in your Markdoc files with the `{% aside %}` tag. Children will be passed to your component's default slot:
 
+```md
+# Welcome to Markdoc 👋
 
-```astro
----
-import { getEntryBySlug } from 'astro:content';
-import Aside from '../components/Aside.astro';
+{% aside type="tip" %}
 
-const entry = await getEntryBySlug('docs', 'why-markdoc');
-const { Content } = await entry.render();
----
+Use tags like this fancy "aside" to add some *flair* to your docs.
```

Comment:
You caught Sarah in the right mood. Flair approved! :nail_care:

Reply:
Thought I could get away with some flair in the example code 😉